js/app/restaurant-week.app.js

```diff
@@ -0,0 +1,10 @@
+(() => {
+	angular
+		.module('rwApp', [])
```

Comment:
Would you spell out restaurantWeekApp rather than using rwApp?

Reply:
talked about this over chat, will need to update the apps naming based on best practices. will do this in a future branch.